qtop.py

```diff
@@ -47,20 +51,23 @@
 #     options.COLORFILE = os.path.expanduser('~/qtop/qtop/qtop.colormap')
 
 
-def colorize(text, pattern, color=None):
+def colorize(text, pattern='Nothing', color_func=None, bg_colour=None):
     """
     prints text coloured according to a unix account pattern color.
+    If color is given, pattern is not needed.
     """
+    # bg_colour = code_of_color['BlueBG']
+    bg_colour = '' if not bg_colour else bg_colour
     try:
-        color_code = code_of_color[color] if color else code_of_color[color_of_account[pattern]]
+        ansi_color = code_of_color[color_func] if color_func else code_of_color[color_of_account[pattern]]
     except KeyError:
         return text
     else:
-        return "\033[" + color_code + "m" + text + "\033[1;m" \
+        return "\033[" + '{}{}'.format(ansi_color, bg_colour) + "m" + text + "\033[0;m" \
             if ((not options.NOCOLOR) and pattern != 'account_not_coloured' and text != ' ') else text
 
 
-def decide_remapping(node_dict):
+def decide_remapping(cluster_dict, _all_letters, _all_str_digits_with_empties):
```

Comment:
not really. Will remove in the future. Put it there mainly to make pycharm stop nagging about shadowing variables